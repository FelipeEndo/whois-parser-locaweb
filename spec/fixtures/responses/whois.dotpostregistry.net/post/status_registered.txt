Domain Name: POSTEITALIANE.POST
Registry Domain ID: D108700000000019482-AGRS
Registrar WHOIS Server:
Registrar URL:
Updated Date: 2021-09-21T22:31:15Z
Creation Date: 2012-09-21T12:03:07Z
Registry Expiry Date: 2022-09-21T12:03:07Z
Registrar Registration Expiration Date:
Registrar: Universal Postal Union
Registrar IANA ID: 9999
Registrar Abuse Contact Email:
Registrar Abuse Contact Phone:
Reseller:
Domain Status: ok https://icann.org/epp#ok
Registry Registrant ID: C19309314-AGRS
Registrant Name: Poste Italiane
Registrant Organization: Poste Italiane
Registrant Street: Viale Europa 190
Registrant City: Rome
Registrant State/Province:
Registrant Postal Code: 00144
Registrant Country: IT
Registrant Phone: +39.0659581
Registrant Phone Ext:
Registrant Fax: +39.065942298
Registrant Fax Ext:
Registrant Email: info@poste.it
Registry Admin ID: C19309312-AGRS
Admin Name: Giovanni Brardinoni
Admin Organization: Poste Italiane
Admin Street: Viale Europa 175
Admin City: Rome
Admin State/Province:
Admin Postal Code: 00144
Admin Country: IT
Admin Phone: +39.0659583671
Admin Phone Ext:
Admin Fax: +39.0698688651
Admin Fax Ext:
Admin Email: Giovanni.Brardinoni@Postecom.it
Registry Tech ID: C19309311-AGRS
Tech Name: Andrea Speranza
Tech Organization: Poste Italiane
Tech Street: Viale Europa 175
Tech City: Rome
Tech State/Province:
Tech Postal Code: 00144
Tech Country: IT
Tech Phone: +39.0659583086
Tech Phone Ext:
Tech Fax: +39.0659582032
Tech Fax Ext:
Tech Email: netsecurity@postecom.it
Name Server: DNS.POSTE.IT
Name Server: DNS2.POSTE.IT
DNSSEC: signedDelegation
URL of the ICANN Whois Inaccuracy Complaint Form: https://www.icann.org/wicf/
>>> Last update of WHOIS database: 2021-11-09T16:19:33Z <<<

For more information on Whois status codes, please visit https://icann.org/epp

Access to WHOIS information is provided to assist persons in determining the contents of a domain name registration record in the registry database. The data in this record is provided by The Registry Operator for informational purposes only, and accuracy is not guaranteed.  This service is intended only for query-based access. You agree that you will use this data only for lawful purposes and that, under no circumstances will you use this data to (a) allow, enable, or otherwise support the transmission by e-mail, telephone, or facsimile of mass unsolicited, commercial advertising or solicitations to entities other than the data recipient's own existing customers; or (b) enable high volume, automated, electronic processes that send queries or data to the systems of Registry Operator, a Registrar, or Afilias except as reasonably necessary to register domain names or modify existing registrations. All rights reserved. Registry Operator reserves the right to modify these terms at any time. By submitting this query, you agree to abide by this policy.
The Registrar of Record identified in this output may have an RDDS service that can be queried for additional information on how to contact the Registrant, Admin, or Tech contact of the queried domain name.
