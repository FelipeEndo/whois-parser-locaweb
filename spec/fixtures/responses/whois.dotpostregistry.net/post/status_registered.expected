#disclaimer
  %s == "Access to WHOIS information is provided to assist persons in determining the contents of a domain name registration record in the registry database. The data in this record is provided by The Registry Operator for informational purposes only, and accuracy is not guaranteed.  This service is intended only for query-based access. You agree that you will use this data only for lawful purposes and that, under no circumstances will you use this data to (a) allow, enable, or otherwise support the transmission by e-mail, telephone, or facsimile of mass unsolicited, commercial advertising or solicitations to entities other than the data recipient's own existing customers; or (b) enable high volume, automated, electronic processes that send queries or data to the systems of Registry Operator, a Registrar, or Afilias except as reasonably necessary to register domain names or modify existing registrations. All rights reserved. Registry Operator reserves the right to modify these terms at any time. By submitting this query, you agree to abide by this policy. The Registrar of Record identified in this output may have an RDDS service that can be queried for additional information on how to contact the Registrant, Admin, or Tech contact of the queried domain name."


#domain
  %s == "posteitaliane.post"

#domain_id
  %s == "D108700000000019482-AGRS"


#status
  %s == ["ok https://icann.org/epp#ok"]

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{2012-09-21 12:03:07 UTC}

#updated_on
  %s %CLASS{time}
  %s %TIME{2021-09-21 22:31:15 UTC}

#expires_on
  %s %CLASS{time}
  %s %TIME{2022-09-21 12:03:07 UTC}


#registrar
  %s %CLASS{registrar}
  %s.id           == "9999"
  %s.name         == "Universal Postal Union"
  %s.url          == ""

#registrant_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_REGISTRANT
  %s[0].id            == "C19309314-AGRS"
  %s[0].name          == "Poste Italiane"
  %s[0].organization  == "Poste Italiane"
  %s[0].address       == "Viale Europa 190"
  %s[0].city          == "Rome"
  %s[0].zip           == "00144"
  %s[0].state         == ""
  %s[0].country_code  == "IT"
  %s[0].phone         == "+39.0659581"
  %s[0].fax           == "+39.065942298"
  %s[0].email         == "info@poste.it"

#admin_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_ADMINISTRATIVE
  %s[0].id            == "C19309312-AGRS"
  %s[0].name          == "Giovanni Brardinoni"
  %s[0].organization  == "Poste Italiane"
  %s[0].address       == "Viale Europa 175"
  %s[0].city          == "Rome"
  %s[0].zip           == "00144"
  %s[0].state         == ""
  %s[0].country_code  == "IT"
  %s[0].phone         == "+39.0659583671"
  %s[0].fax           == "+39.0698688651"
  %s[0].email         == "Giovanni.Brardinoni@Postecom.it"

#technical_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_TECHNICAL
  %s[0].id            == "C19309311-AGRS"
  %s[0].name          == "Andrea Speranza"
  %s[0].organization  == "Poste Italiane"
  %s[0].address       == "Viale Europa 175"
  %s[0].city          == "Rome"
  %s[0].zip           == "00144"
  %s[0].state         == ""
  %s[0].country_code  == "IT"
  %s[0].phone         == "+39.0659583086"
  %s[0].fax           == "+39.0659582032"
  %s[0].email         == "netsecurity@postecom.it"


#nameservers
  %s %CLASS{array}
  %s %SIZE{2}
  %s[0] %CLASS{nameserver}
  %s[0].name == "dns.poste.it"
  %s[1] %CLASS{nameserver}
  %s[1].name == "dns2.poste.it"
